scaling_performance/using_cluster_loader.adoc

```diff
@@ -0,0 +1,259 @@
+
+----
+$ git clone https://github.com/openshift/origin.git
+$ cd origin
+$ make build-extended-test
+----
+
+After compilation, find the output binary in the subfolder
+*_./_output/local/bin/linux/amd64`_*.
+
+[[running-cluster-loader]]
+== Running Cluster Loader
+
+. Set the `KUBECONFIG` variable to the location of the administrator `kubeconfig`:
++
+----
+$ export KUBECONFIG=${KUBECONFIG-$HOME/.kube/config}
+----
+
+. Execute Cluster Loader using the built-in test configuration, which deploys five
+template builds and waits for them to complete.
++
+----
+$ cd /usr/libexec/atomic-openshift/
+./extended.test --ginkgo.focus="Load cluster"
+----
++
+Alternatively, execute Cluster Loader with a user-defined configuration by
+adding the flag for `--viper-config`:
++
+----
+$ ./extended.test --ginkgo.focus="Load cluster" --viper-config=config/test <1>
+----
+<1> In this example, there is a subdirectory called `config` with a configuration
+file called *_test.yml_*. In the command line, exclude the extension of the
+configuration file, as the tool will automatically determine the file type and
+extension.
+
+[[configuring-cluster-loader]]
+== Configuring Cluster Loader
+
+Create multiple namespaces (projects), which contain multiple templates or pods.
+
+Locate the configuration files for Cluster Loader in the *_config/_*
+subdirectory. The pod files and template files referenced in these configuration
+examples are found in the *_content/_* subdirectory.
+
+[[configuring-cluster-loader-configuration-fields]]
+=== Configuration Fields
+
+.Top-level Cluster Loader Fields
+|===
+|Field |Description
+
+|`cleanup`
+|Set to `true` or `false`. One definition per configuration. If set to `true`,
+`cleanup` will delete all namespaces (projects) created by Cluster Loader at the
+end of the test.
+
+|`projects`
+|A sub-object with one or many definition(s). Under `projects`, each
+namespace to create is defined and `projects` has several mandatory subheadings.
+
+|`tuningSet`
+|A sub-object with one definition per configuration. `TuningSet` allows the user
+to define a tuning set to add configurable timing to project or object creation
+(pods, templates, etc).
+|===
+
+.Fields under `projects`
+|===
+|Field |Description
+
+|`num`
+|Number, an integer. One definition of the count of how many projects to create.
+
+|`basename`
+|A string. One definition of the base name for the project. The count of
+identical namespaces will be appended to `Basename` to prevent collisions.
+
+|`tuning`
+|A string. One definition of what tuning set you want to apply to the objects,
+which you deploy inside this namespace.
+
+|`pods`
+|A sub-object with one or many definition(s) of pods to deploy.
+
+|`templates`
+|A sub-object with one or many definition(s) of templates to deploy.
+|===
+
+.Fields under `pods` and `templates`
+|===
+|Field |Description
+
+|`total`
+|This field is not used.
+
+|`num`
+|An integer. The number  of pods or templates to deploy.
```

Comment:
num" works only for pods. Looking at code sugests it wasn't implemented for templates. @sjug want an issue or is this intentional and this doc should be updated?

Reply:
@jmencak Thanks! I think that was a typo on my part. Source content from @sjug says number.

Reply:
Nice spot @jmencak that is correct, we're only deploying one template per namespace.